libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/RadioButton.kt

```diff
@@ -63,18 +63,17 @@ internal fun compoundRadioButtonColors(): RadioButtonColors {
 
 @Preview(group = PreviewGroup.Toggles)
 @Composable
-internal fun RadioButtonPreview() = ElementThemedPreview(vertical = false) { ContentToPreview() }
+internal fun RadioButtonCheckedPreview() = ElementThemedPreview(vertical = false) { ContentToPreview(true) }
+
+@Preview(group = PreviewGroup.Toggles)
+@Composable
+internal fun RadioButtonUncheckedPreview() = ElementThemedPreview(vertical = false) { ContentToPreview(false) }
 
 @Composable
-private fun ContentToPreview() {
+private fun ContentToPreview(defaultCheck: Boolean) {
+    var checked by remember { mutableStateOf(defaultCheck) }
     Column {
-        Row(horizontalArrangement = Arrangement.spacedBy(6.dp)) {
-            RadioButton(selected = false, onClick = {})
-            RadioButton(selected = false, enabled = false, onClick = {})
-        }
-        Row(horizontalArrangement = Arrangement.spacedBy(6.dp)) {
```

Comment:
Actually, these were used so the designers could take a quick look in Showkase and verify the buttons look good in all possible states by looking at a single preview. Could we go back to displaying the 4 states at the same time?

If we want the interaction to work, we could use `selected = checked` in the first 2 components and `selected = !checked` in the next two?

Reply:
Done, I have an issue with the interactive mode on the `RadioButton` preview but the interaction works fine in Showkase

Reply:
(I just triggered the screenshots action)